diff --git a/dist/ember.debug.js b/dist/ember.debug.js
index 58e681ac01f6dd9d202041bdabcec15ae4d3faea..f847880a662edfcaf0c64546dd37fffeaad0d3ca 100644
--- a/dist/ember.debug.js
+++ b/dist/ember.debug.js
@@ -10221,7 +10221,7 @@ var define, require;
             the subsequent cache invalidations of the `@cached` properties who were
             using this `trackedProp`.
 
-            Remember that setting tracked data should only be done during initialization, 
+            Remember that setting tracked data should only be done during initialization,
             or as the result of a user action. Setting tracked data during render
             (such as in a getter), is not supported.
 
@@ -14292,25 +14292,25 @@ var define, require;
             /**
               `catch` is simply sugar for `then(undefined, onRejection)` which makes it the same
               as the catch block of a try/catch statement.
-            
+
               ```js
               function findAuthor(){
                 throw new Error('couldn\'t find that author');
               }
-            
+
               // synchronous
               try {
                 findAuthor();
               } catch(reason) {
                 // something went wrong
               }
-            
+
               // async with promises
               findAuthor().catch(function(reason){
                 // something went wrong
               });
               ```
-            
+
               @method catch
               @param {Function} onRejection
               @param {String} [label] optional string for labeling the promise.
@@ -14324,9 +14324,9 @@ var define, require;
             /**
               `finally` will be invoked regardless of the promise's fate just as native
               try/catch/finally behaves
-            
+
               Synchronous example:
-            
+
               ```js
               findAuthor() {
                 if (Math.random() > 0.5) {
@@ -14334,7 +14334,7 @@ var define, require;
                 }
                 return new Author();
               }
-            
+
               try {
                 return findAuthor(); // succeed or fail
               } catch(error) {
@@ -14344,9 +14344,9 @@ var define, require;
                 // doesn't affect the return value
               }
               ```
-            
+
               Asynchronous example:
-            
+
               ```js
               findAuthor().catch(function(reason){
                 return findOtherAuthor();
@@ -14354,7 +14354,7 @@ var define, require;
                 // author was either found, or not
               });
               ```
-            
+
               @method finally
               @param {Function} callback
               @param {String} [label] optional string for labeling the promise.
@@ -39439,7 +39439,7 @@ var define, require;
                   nextSibling: null
                 }), layout, dynamicScope);
                 let result = this.result = iterator.sync();
-                associateDestroyableChild(owner, result);
+                associateDestroyableChild(this, result);
                 this.render = errorLoopTransaction(() => {
                   if (isDestroying(result) || isDestroyed(result)) return;
                   return result.rerender({
diff --git a/dist/ember.prod.js b/dist/ember.prod.js
index 7e3f801442ce6dcf77caf7dbc0d69c6a838877cb..4c4fd5ed4d9f92cfa532b3ffa80ec8036570b401 100644
--- a/dist/ember.prod.js
+++ b/dist/ember.prod.js
@@ -8921,7 +8921,7 @@ var define, require;
             the subsequent cache invalidations of the `@cached` properties who were
             using this `trackedProp`.
 
-            Remember that setting tracked data should only be done during initialization, 
+            Remember that setting tracked data should only be done during initialization,
             or as the result of a user action. Setting tracked data during render
             (such as in a getter), is not supported.
 
@@ -12889,25 +12889,25 @@ var define, require;
             /**
               `catch` is simply sugar for `then(undefined, onRejection)` which makes it the same
               as the catch block of a try/catch statement.
-            
+
               ```js
               function findAuthor(){
                 throw new Error('couldn\'t find that author');
               }
-            
+
               // synchronous
               try {
                 findAuthor();
               } catch(reason) {
                 // something went wrong
               }
-            
+
               // async with promises
               findAuthor().catch(function(reason){
                 // something went wrong
               });
               ```
-            
+
               @method catch
               @param {Function} onRejection
               @param {String} [label] optional string for labeling the promise.
@@ -12921,9 +12921,9 @@ var define, require;
             /**
               `finally` will be invoked regardless of the promise's fate just as native
               try/catch/finally behaves
-            
+
               Synchronous example:
-            
+
               ```js
               findAuthor() {
                 if (Math.random() > 0.5) {
@@ -12931,7 +12931,7 @@ var define, require;
                 }
                 return new Author();
               }
-            
+
               try {
                 return findAuthor(); // succeed or fail
               } catch(error) {
@@ -12941,9 +12941,9 @@ var define, require;
                 // doesn't affect the return value
               }
               ```
-            
+
               Asynchronous example:
-            
+
               ```js
               findAuthor().catch(function(reason){
                 return findOtherAuthor();
@@ -12951,7 +12951,7 @@ var define, require;
                 // author was either found, or not
               });
               ```
-            
+
               @method finally
               @param {Function} callback
               @param {String} [label] optional string for labeling the promise.
@@ -36989,7 +36989,7 @@ var define, require;
                   nextSibling: null
                 }), layout, dynamicScope);
                 let result = this.result = iterator.sync();
-                associateDestroyableChild(owner, result);
+                associateDestroyableChild(this, result);
                 this.render = errorLoopTransaction(() => {
                   if (isDestroying(result) || isDestroyed(result)) return;
                   return result.rerender({
diff --git a/dist/packages/shared-chunks/index-DaCZcoCY.js b/dist/packages/shared-chunks/index-DaCZcoCY.js
index dc6321247b9fa08737eef218fcfb520853c312ee..17ac4d3d9eefd4b703ee78e7882bdab775a0c560 100644
--- a/dist/packages/shared-chunks/index-DaCZcoCY.js
+++ b/dist/packages/shared-chunks/index-DaCZcoCY.js
@@ -5205,7 +5205,7 @@ class ClassicRootState {
         nextSibling: null
       }), layout, dynamicScope);
       let result = this.result = iterator.sync();
-      associateDestroyableChild(owner, result);
+      associateDestroyableChild(this, result);
       this.render = errorLoopTransaction(() => {
         if (isDestroying(result) || isDestroyed(result)) return;
         return result.rerender({
