diff --git a/node_modules/fastboot/src/fastboot-info.js b/node_modules/fastboot/src/fastboot-info.js
index 50b181b..f330596 100644
--- a/node_modules/fastboot/src/fastboot-info.js
+++ b/node_modules/fastboot/src/fastboot-info.js
@@ -38,6 +38,5 @@ module.exports = class FastBootInfo {
    */
   register(instance) {
     instance.register('info:-fastboot', this, { instantiate: false });
-    instance.inject('service:fastboot', '_fastbootInfo', 'info:-fastboot');
   }
 };
